recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/gammasimtools-{{ version }}.tar.gz
-  sha256: 4f297f285b4d9d9f8dd31d3944c6b15073883b6416e6ac12802f9c6d1c6642fa
+  sha256: a51844b731f20896e58ee86eff06e3bf1aec4b0b4077a3d9f8009e5c74ddd197
```

Comment:
You also need to revert back the hash, not just the version.

Reply:
Yes yes, sorry, I made a terrible mess with this learning how to use this system while doing other things.